tests/pcmjava/tools.vitruv.applications.pcmjava.tests.util/src/tools/vitruv/applications/pcmjava/tests/util/Java2PcmTransformationTest.java

```diff
@@ -127,7 +127,7 @@
 
 	private static final Logger logger = Logger.getLogger(Java2PcmTransformationTest.class.getSimpleName());
 
-	private static int MAXIMUM_SYNC_WAITING_TIME = 10000;
+	private static int MAXIMUM_SYNC_WAITING_TIME = 15000;
```

Comment:
Do you really have situations in which waiting for 10 seconds is not enough?

Reply:
Yes, when running the tests on my machine, the first test case would sometimes fail with a timeout.

Reply:
Alright, then let's have the time increased. I just wanted to make sure that it was not some local testing commited by accident, but actually you made an explicit commit for that, so that was an unreasonable assumption by me.

Reply:
Still, better to talk about it than to have a worse codebase!